packages/agents-hosting/src/oauth/userTokenClient.ts

```diff
@@ -81,24 +81,24 @@ export class UserTokenClient {
 
   /**
    * Gets the sign-in resource.
-   * @param appId The application ID.
-   * @param cnxName The connection name.
+   * @param msAppId The application ID.
+   * @param connectionName The connection name.
    * @param activity The activity.
    * @returns A promise that resolves to the signing resource.
    */
-  async getSignInResource (appId: string, cnxName: string, conversationReference: ConversationReference, relatesTo?: ConversationReference) : Promise<SigningResource> {
+  async getSignInResource (msAppId: string, connectionName: string, conversation: ConversationReference, relatesTo?: ConversationReference) : Promise<SignInResource> {
```

Comment:
@MattB-msft do we need to keep this method?

Reply:
lets keep this for now until we are fully confident in the global deployment of the new process, 
tag this with an obsoleet / warning its being removed.